maistra/test-with-asan.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# This script will execute Envoy's c++ unit and integration tests with
+# address sanitizer enabled. It may point out issues related to both
+# undefined behaviour as well as memory management issues (leaks).
+
+# An argument can be passed to this script which when specified will replace
+# the "//test/..." part we pass to bazel. This allows one to optionally
+# specify which tests to run.
+
+# Furthermore, while this defaults to asan, one can set the SANITIZER env var
+# to specify other sanitizers, as support in our bazel setup. 
+# As of writing this, known valid values are "tsan" and "msan".
+
+# Lastly, when running this in development flows, one might want to set BAZEL_CONFIG=
```

Comment:
`s/BAZEL_CONFIG/CI/` ?

Reply:
Fixed in 63e6ee7051427382c107bc6e7621530c579d2eed